src/site/asciidoc/index.adoc

```diff
@@ -117,9 +117,9 @@ See link:manual/cloud.html[Logging in the Cloud] for details.
 
 === Compatible with Log4j 1.x
 
-The Log4j-1.2-api module of Log4j 2 provides compatiblity for applications using the Log4j 1 logging methods. As
+The Log4j-1.2-api module of Log4j 2 provides compatibility for applications using the Log4j 1 logging methods. As
 of Log4j 2.13.0 Log4j 2 also provides experimental support for Log4j 1.x configuration files. See
-link:manual/compatiblity.html[Log4j 2 Compatiblity with Log4j 1] for more information.
+link:manual/compatibility.html[Log4j 2 Compatibility with Log4j 1] for more information.
```

Comment:
This developer facing link is bad, as the actual file is: src/site/markdown/manual/compatibility.md